### ISSUE TITLE
Breaking change: verdaccio 5.5.0 requires node 14, but documentation says verdaccio 5 requires node 12

### ISSUE THREAD

**Issue by marvinthepa** [verdaccio/verdaccio#2963]
**Your Environment**
 * **verdaccio version**: 5.5.0
 * **node version** 12.22.6
 * **package manager**: yarn 1.22.4
 * **os**: OpenBSD
 * **platform**: npm, docker

**Describe the bug**

When yarn installing verdaccio, it complains about the node engine version required by its dependency  @verdaccio/ui-theme (see below).

**To Reproduce**

```
$ yarn upgrade verdaccio@5.5.0
yarn upgrade v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...
error @verdaccio/ui-theme@6.0.0-6-next.15: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.6"
error Found incompatible module.
```

**Expected behavior**

Either
* the stable release of verdaccio should not depend on new verdaccio/ui-theme versions that require node 14, or
* Documentation should be updated that verdaccio now requires node 14

It seems that this was introduced in #2473. With d36b8b1e00ed418042, "Only affect v6.x alpha" does not seem to be true anymore, as 5.5.0 is affected as well.

**Comment by juanpicado** [comment]
Sorry my bad 😞, I didn't  check that. Thanks for reporting.

**Comment by juanpicado** [comment]
Solved v5.5.1 👌🏼
